addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2868,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#670"
-msgid "Verbose logging for CURL library (http, dav)"
+msgid "Verbose logging for \"libcURL\" library \"HTTP(S), DAV\""
```

Comment:
`HTTP(S), DAV` should be inside parentheses, they serve as an example of what the CURL library is used for.